### PR TITLE
Feature/add projectname query param

### PR DESCRIPTION
**Pull request by FredrikOseberg**
Implements support for adding project name as query parameter on the unleash clients polling. 

[x] added support
[x] unit tests
[x] client tests

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -10,5 +10,33 @@ github.com/stretchr/testify v1.2.2 h1:bSDNvY7ZPG5RlJ8otE/7V6gMiyenm9RtJ7IUVIAoJ1
 github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXfy6kDkUVs=
 github.com/twmb/murmur3 v1.1.5 h1:i9OLS9fkuLzBXjt6dptlAEyk58fJsSTXbRg3SgVyqgk=
 github.com/twmb/murmur3 v1.1.5/go.mod h1:Qq/R7NUyOfr65zD+6Q5IHKsJLwP7exErjN6lyyq3OSQ=
+github.com/yuin/goldmark v1.2.1/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
```
Comment (ivarconr) [inline go.sum:13]: What is all this?

Comment (FredrikOseberg) [reply go.sum:13]: Just noticed it. I suspect it's artifacts from installing vscode gotools. Should be removed.

Comment (FredrikOseberg) [reply go.sum]: Forgot to run ```go mod tidy```.

**Review by ivarconr — APPROVED** [review]

**Review by jrbarron — APPROVED** [review]
Looks good! :+1:

**Pull request merged**